src/MainWindow.vala

```diff
@@ -271,6 +274,46 @@ public class Tasks.MainWindow : Hdy.ApplicationWindow {
         });
     }
 
+    private void dialog_refresh_collection_error (Error e) {
+        string error_message = e.message;
+
+        GLib.Idle.add (() => {
+            var error_dialog = new Granite.MessageDialog (
+                _("Refresh collection failed"),
+                _("The backend may be unavailable or unable to read from."),
+                new ThemedIcon ("dialog-error"),
```

Comment:
Maybe this should `preferences-desktop-online-accounts` with a badge of `dialog-error` to make it clearer this is an issue with getting data from somewhere else?